src/site/content/en/blog/handwriting-recognition/index.md

```diff
@@ -0,0 +1,426 @@
+authors:
+  - christianliebel
+  - thomassteiner
+
+date: 2021-05-14
+# updated: 2021-05-14
+hero: image/8WbTDNrhLsU0El80frMBGE4eMCD3/k1gdvpBMneFVrOC5h4yQ.jpg
+alt: A hand draws letters in calligraphic script on a paper.
+tags:
+  - blog # blog is a required tag for the article to show up in the blog.
+  - capabilities
+
+origin_trial:
+  url: https://developer.chrome.com/origintrials/#/view_trial/🎉TODO_UPDATE🎉
+---
+
+{% Aside %} The Handwriting Recognition API is part of the [capabilities project](/fugu-status/) and
+is currently in development. This post will be updated as the implementation progresses.
+{% endAside %}
+
+## What is the Handwriting Recognition API? {: #what }
+
+> TODO: GIF of demo
+
+The Handwriting Recognition API is a new web platform API that allows you to convert handwritten
+drawings (ink) from your users into text. Some operating systems have long included such APIs,
+and with this new capability, your web apps can finally leverage the same powerful functionality.
+The conversion takes place directly on the user's device, works even in offline mode, all without
+adding any third-party libraries or services.
+
+This API implements a so-called "on-line" or real-time recognition. This means that the handwritten
+input is recognized while the user is drawing it, by capturing and analyzing the single strokes. In
+contrast to "off-line" procedures such as Optical Character Recognition (OCR), where only the end
+product is known, on-line algorithms can provide a higher level of accuracy.
+
+### Suggested use cases for the Handwriting Recognition API {: #use-cases }
+
+Examples of sites that may use this API include:
+
+- Note-taking applications where users want to capture handwritten notes and have them translated
+  into text.
+- Forms applications where users use pen or finger input due to time constraints.
+- Games that require filling in letters, such as crosswords or Hangman.
+
+## Current status {: #status }
+
+<div class="w-table-wrapper">
+
+| Step                                     | Status                   |
+| ---------------------------------------- | ------------------------ |
+| 1. Create explainer                      | [Complete][explainer]    |
+| 2. Create initial draft of specification | Not started              |
+| 3. Gather feedback & iterate on design   | [In progress](#feedback) |
+| 4. Origin trial                          | Not started              |
+| 5. Launch                                | Not started              |
+
+</div>
+
+## How to use Handwriting Recognition {: #use }
+
+### Enabling via chrome://flags
+
+To experiment with the Handwriting Recognition API locally, without an origin trial token, enable
+the `#experimental-web-platform-features` flag in `chrome://flags`.
+
+{% Aside %} Note that the API is currently exclusive to Chrome&nbsp;OS devices. {% endAside %}
+
+> REVIEW: Add the version? Probably M90+
+
+> REVIEW: There is no Origin Trial yet.
```

Comment:
@tomayac What about those?

Reply:
The link will come… Watch the bug.

Reply:
Or we launch the article with just a Dev Trial.

Reply:
Ah, I didn't know OT is basically around the corner (didn't have access to the issue before). I watch the bug now.

Reply:
See https://groups.google.com/a/chromium.org/d/msgid/blink-dev/CANFkJJmGJVmkrO1xAeDNc9i9%2Bp5z%2BacpRPDAD8doxYnFGN37%3Dg%40mail.gmail.com

_"Due to technical reasons, we plan to run an origin trial in M91 on a limited set of devices that can support it._

_We plan to expand the origin trial coverage to all Chrome OS devices in M92, and run a Chrome-OS-wide origin trial from M92 to M94."_

--
Unrelated PS: Other links worth adding (if you don't have them yet):

- https://github.com/mozilla/standards-positions/issues/507
- https://lists.webkit.org/pipermail/webkit-dev/2021-March/031762.html
- https://groups.google.com/a/chromium.org/d/msgid/blink-dev/CANFkJJmpn2FtAXcWgYGvz04_DVBkkkiXXHvSM7jg5haH3ZvRJQ%40mail.gmail.com